lib/sql_string.js

```diff
@@ -130,11 +122,11 @@ function zeroPad(number, length) {
 }
 
 function convertTimezone(tz) {
-  if (tz == "Z") return 0;
+  if (tz === "Z") return 0;
 
   var m = tz.match(/([\+\-\s])(\d\d):?(\d\d)?/);
   if (m) {
-    return (m[1] == '-' ? -1 : 1) * (parseInt(m[2], 10) + ((m[3] ? parseInt(m[3], 10) : 0) / 60)) * 60;
+    return (m[1] === '-' ? -1 : 1) * (parseInt(m[2], 10) * 60 + (m[3] ? parseInt(m[3], 10) : 0));
   }
-  return false;
+  return 0;
```

Comment:
This does not look right to me. Whats the reason for returning `0` in case `tz` is "Z" or `tz.match` does not match anything? I am guessing you have done it to hide `consistent-return` ESLint error. Though, that warning is not saying that you should try to make the returning types match – it is warning about a potentially dangerous pattern.

In this case, I think that this function should throw an error in all cases that it is now returning `0`. This further implies that the calling context must be aware of these limitations and must not call the function in situations where it would cause error.